src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -2001,6 +2001,8 @@ public abstract class DbBatchCommand
         public abstract int RecordsAffected { get; }
         public System.Data.Common.DbParameterCollection Parameters { get { throw null; } }
         protected abstract System.Data.Common.DbParameterCollection DbParameterCollection { get; }
+        public virtual DbParameter CreateParameter() { throw null; }
```

Comment:
Everything else seems to be fully qualified, isn't it needed here?
```c#
public virtual System.Data.Common.DbParameter CreateParameter() { throw null; }
```

Reply:
Thanks - #89775